src/main/java/com/exasol/adapter/RequestDispatcher.java

```diff
@@ -1,34 +1,24 @@
 package com.exasol.adapter;
 
+import java.util.*;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import com.exasol.ExaMetadata;
-import com.exasol.adapter.request.*;
+import com.exasol.adapter.request.AdapterRequest;
+import com.exasol.adapter.request.LoggingConfiguration;
 import com.exasol.adapter.request.parser.RequestParser;
-import com.exasol.adapter.response.*;
-import com.exasol.adapter.response.converter.ResponseJsonConverter;
 import com.exasol.errorreporting.ExaError;
 import com.exasol.logging.RemoteLogManager;
 import com.exasol.logging.VersionCollector;
 
 /**
- * This class is the main entry point for calls to a Virtual Schema. From here the adapter calls are dispatched to the
- * responsible adapter.
+ * This class is the main entry point for calls to a Virtual Schema. It sets up the application and delegate the control
+ * to the {@link AdapterCallExecutor}.
  */
 public final class RequestDispatcher {
```

Comment:
This class does no longer dispatch. But probably you don't want to rename it since it is the entrypoint ,right?

Reply:
Exactly, the plan is to rename it to something more meaningful but this class is used outside, so I can't do it at the moment.